crates/solver/src/main.rs

```diff
@@ -280,6 +280,19 @@ struct Arguments {
     /// but at the same time we don't restrict solutions sizes too much
     #[clap(long, env, default_value = "15000000")]
     simulation_gas_limit: u128,
+
+    /// In order to protect against malicious solvers, the driver will check that settlements prices do not
+    /// exceed a max price deviation compared to the external prices of the driver, if this optional value is set.
+    /// The max deviation value should be provided as a percentage value. E.g. for a max price deviation
+    /// of 3%, one should set it to 3u64
+    #[clap(long, env)]
+    max_settlement_price_deviation: Option<u64>,
+
+    /// This variable allows to restrict the set of tokens for which a price deviation check of settlement
+    /// prices and external prices is executed. If the value is set to none, then all tokens included
```

Comment:
> If the value is set to none

From the perspective of someone using the CLI this wording would be confusing because they can't set the value to `None` expliclitly.